### PR TITLE
chore(macros): update zh-CN strings in CSS

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
only sync with English versions, not uniform with title and sidebar.

### CODE REVIEW

**Comment by awxiaoxian2020** [comment]
@yin1999 I don't know why there are some `undefined` in my local env.
issue

**Comment by yin1999** [comment]
> @yin1999 I don't know why there are some `undefined` in my local env. issue

See line 1041 and line 1073:

```diff
-            <li><%-smartLink(`${cssURL}CSS_Grid_Layout/CSS_Grid_Logical_Values_and_Writing_Modes`, null, text['Grids,_logical_values_and_writing_modes'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_Grid_Layout/CSS_Grid_Logical_Values_and_Writing_Modes`, null, text['Grids_logical_values_and_writing_modes'], cssURL)%></li>
```

```diff
-            <li><%-smartLink(`${cssURL}CSS_Logical_Properties/Margins_borders_padding`, null, text['Margins,_borders_and_padding'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_Logical_Properties/Margins_borders_padding`, null, text['Margins_borders_and_padding'], cssURL)%></li>
```

File: kumascript/macros/CSSRef.ejs [hunk 1]
```diff
@@ -577,48 +577,151 @@ const text = mdn.localStringMap({
...
+      'Conditional_rules': '条件规则',
+        'Using_feature_queries': '运用查询特性',
+      'CSSOM_view': 'CSS 对象模型视图',
+        'Coordinate_systems': '坐标系',
+      'Flexbox': 'Flex 布局',
+        'Basic_concepts_of_Flexbox': 'Flex 布局基础',
+        'Comparison_with_other_layout_methods': '与其他布局的比较',
+        'Aligning_items_in_a_flex_container': 'Flex 容器中的对齐方式',
+        'Ordering_flex_items': 'Flex 布局中的排序方式',
+        'Controlling_flex_item_ratios': '控制 Flex 布局中元素的比例',
```
Comment (yin1999) [inline kumascript/macros/CSSRef.ejs:653]:
```suggestion
        'Controlling_flex_item_ratios': '控制 Flex 子元素在主轴上的比例',
```

File: kumascript/macros/CSSRef.ejs [hunk 2]
```diff
@@ -577,48 +577,151 @@ const text = mdn.localStringMap({
...
+        'Mastering_wrapping_of_flex_items': '包装 Flex 布局中的元素',
+        'Typical_use_cases_of_Flexbox': '经典的 Flex 布局示例',
+        'Backwards_compatibility_of_Flexbox': 'Flex 布局中的兼容性',
```
Comment (yin1999) [inline kumascript/macros/CSSRef.ejs:656]:
```suggestion
        'Backwards_compatibility_of_Flexbox': 'Flex 布局的向下兼容性',
```

File: kumascript/macros/CSSRef.ejs [hunk 3]
```diff
@@ -577,48 +577,151 @@ const text = mdn.localStringMap({
...
+      'Flow_layout': '流布局',
+        'Block_and_Inline_layout_in_normal_flow': '一般的流布局中的块和行布局',
+        'In_flow_and_Out_of_flow': '流布局的内外',
```
Comment (yin1999) [inline kumascript/macros/CSSRef.ejs:659]:
```suggestion
        'Block_and_Inline_layout_in_normal_flow': '一般的流布局中的块和内联布局',
        'In_flow_and_Out_of_flow': '应用或脱离流布局',
```

File: kumascript/macros/CSSRef.ejs [hunk 4]
```diff
@@ -577,48 +577,151 @@ const text = mdn.localStringMap({
...
+        'Formatting_contexts_explained': '格式化上下文简介',
+        'Flow_layout_and_writing_modes': '流布局和书写模式',
+        'Flow_layout_and_overflow': '流布局和溢出',
+      'Fonts': '字体',
+        'OpenType_font_features_guide': 'OpenType 字体特点',
```
Comment (yin1999) [inline kumascript/macros/CSSRef.ejs:664]:
```suggestion
        'OpenType_font_features_guide': 'OpenType 字体特性指南',
```

File: kumascript/macros/CSSRef.ejs [hunk 5]
```diff
@@ -577,48 +577,151 @@ const text = mdn.localStringMap({
...
+        'Auto-placement_in_grid_layout': 'Grid 布局中的自动定位',
+        'Box_alignment_in_grid_layout': 'Grid 布局中的盒模型对齐',
+        'Grids_logical_values_and_writing_modes': '网格、逻辑值和书写模式',
+        'Grid_layout_and_accessibility': 'Grid 布局和无障碍',
+        'Grid_Layout_and_progressive_enhancement': 'Grid 布局和渐进增强',
+        'Realizing_common_layouts_using_grids': '运用 Grid 布局实现常见布局',
+        'Subgrid': '子网格',
+        'Masonry_layout': 'Masonry 布局',
+      'Images': '图片',
+        'Using_CSS_gradients': '运用 CSS 渐变特性',
```
Comment (yin1999) [inline kumascript/macros/CSSRef.ejs:681]:
```suggestion
        'Using_CSS_gradients': '使用 CSS 渐变',
```

File: kumascript/macros/CSSRef.ejs [hunk 6]
```diff
@@ -577,48 +577,151 @@ const text = mdn.localStringMap({
...
+      'Lists_and_counters': '列表和缩进',
```
Comment (yin1999) [inline kumascript/macros/CSSRef.ejs:682]:
```suggestion
      'Lists_and_counters': '列表和计数器',
```

File: kumascript/macros/CSSRef.ejs [hunk 7]
```diff
@@ -577,48 +577,151 @@ const text = mdn.localStringMap({
...
+        'Using_CSS_counters': '运用 CSS 计数器',
+        'Consistent_list_indentation': '调整列表缩进',
+      'Logical_properties': '逻辑属性',
+        'Basic_concepts': '基础概念',
+        'Floating_and_positioning': '浮动和定位',
+        'Margins_borders_padding': '内外边距和边框',
+        'Sizing': '尺寸',
```
Comment (yin1999) [inline kumascript/macros/CSSRef.ejs:689]:
```suggestion
        'Sizing': '调整尺寸',
```

File: kumascript/macros/CSSRef.ejs [hunk 8]
```diff
@@ -577,48 +577,151 @@ const text = mdn.localStringMap({
...
+      'Scroll_snap': 'Scroll snap',
+        'Basic_concepts_of_scroll_snap': 'scroll snap 基础概念',
+        'Browser_compatibility_and_scroll_snap': 'scroll snap 和浏览器兼容性',
+      'Shapes': '形状',      
+        'Overview_of_shapes': '形状概述',
+        'Shapes_from_box_values': '使用 box 值指定形状',
+        'Basic_shapes': '基础的形状',
+        'Shapes_from_images': '从图片中指定形状',
+      'Text': '文本',
+        'Wrapping_and_breaking_text': '包装文本和换行文本',
```
Comment (yin1999) [inline kumascript/macros/CSSRef.ejs:705]:
```suggestion
        'Wrapping_and_breaking_text': '包装文本和文本断行',
```

File: kumascript/macros/LearnSidebar.ejs [hunk 9]
```diff
@@ -2316,4 +2317,4 @@ var text = mdn.localStringMap({
   </li>
 </ol>
 
-</section>
+</section>
```
Comment (yin1999) [inline kumascript/macros/LearnSidebar.ejs:2320]: add a newline here:

```suggestion
</section>

```

**Review by yin1999 — COMMENTED** [review]
some suggestions

**Comment by yin1999** [comment]
LGTM, poke @mdn/mdn-community-engagement for a review. Please take care of line 1041 and 1073 in `kumascript/macros/CSSRef.ejs`.

**Review by schalkneethling — APPROVED** [review]
Thank you, @awxiaoxian2020 and @yin1999  🎉

**Pull request merged**